tensorboard/webapp/widgets/data_table/data_table_test.ts

```diff
@@ -113,60 +113,12 @@ describe('data table', () => {
           displayName: 'Run',
           enabled: true,
         },
-        {
-          type: ColumnHeaderType.STEP,
-          name: 'step',
-          displayName: 'Step',
-          enabled: true,
-        },
-        {
-          type: ColumnHeaderType.RELATIVE_TIME,
-          name: 'relativeTime',
-          displayName: 'Relative',
-          enabled: true,
-        },
```

Comment:
Is there a reason you are removing these from the mocks?

Reply:
No need to test that all of these display the correct text as all it does now is display the displayText. We do not need to test that it displays the displayText that many times.